server/http.go

```diff
@@ -16,20 +16,23 @@ import (
 	"github.com/avenga/couper/config/env"
 	"github.com/avenga/couper/config/request"
 	"github.com/avenga/couper/config/runtime"
+	"github.com/avenga/couper/errors"
 	"github.com/avenga/couper/eval"
 	"github.com/avenga/couper/handler"
 	"github.com/avenga/couper/handler/transport"
 	"github.com/avenga/couper/logging"
 )
 
+type muxes map[string]*Mux
+
 // HTTPServer represents a configured HTTP server.
 type HTTPServer struct {
 	accessLog  *logging.AccessLog
 	commandCtx context.Context
 	evalCtx    *eval.Context
 	listener   net.Listener
 	log        logrus.FieldLogger
-	mux        *Mux
+	muxes      muxes
```

Comment:
Can we find a better wording? Mux is the short form for one multiplexer;  more multiplexers.
Therefore `muxers` ?

Reply:
Done